src/cli.rs

```diff
@@ -53,6 +57,9 @@ $ nix-template config nixpkgs-root ~/nixpkgs
                 .default_value_if("TEMPLATE", Some("test"), "test.nix")
                 .default_value_if("TEMPLATE", Some("flake"), "flake.nix"),
         )
+        .arg(Arg::from_usage(
+            "-u,--from-url [url] 'Point to a github repo, and use github api to determine package values'",
```

Comment:
Should this mention that it uses GITHUB_TOKEN env to make authenticated requests?

Reply:
you get ~100 requests per hour, but yes, it should

Reply:
I think I am a special exception here but I depending on the type and what is happening I sometimes actually hit the api limit for some requests.